tests/specs/type_tracing/TypeScriptTypesHeader.txt

```diff
@@ -0,0 +1,314 @@
+# mod.ts
```

Comment:
This new test actually has incorrect output. Just ignore it for now. I'll follow up in a separate PR.